frontend/packages/kubevirt-plugin/src/constants/storage-class.tsx

```diff
@@ -0,0 +1,2 @@
+export const STORAGE_CLASS_CONFIG_MAP_NAME = 'storage-class-defaults';
```

Comment:
Since this is kubevirt-specific I think we should name this kubevirt-storage-class-defaults

Reply:
done